book/website/community-handbook/content.md

```diff
@@ -0,0 +1,139 @@
+
+### Chapters
+
+Chapters are sub-folders within Guides.
+
+For example, the [Version Control](https://the-turing-way.netlify.app/reproducible-research/vcs.html) chapter in the Guide for Reproducible Research has the following structure:
+
+```
+book\website
+│
+└───reproducible-research <---- (folder for the Guide to Reproducible Research)
+│   │   reproducible-research.md ---- (Guide's Landing Page)
+│   │   vcs.md <---- (Landing page for the Version Control chapter)
+│   │   example-chapter.md (landing page for a new chapter)
+|   |
+│   └───vcs (chapter folder)
+│   |   │   vcs-workflow.md
+|   |   |   vcs-git.md
+|   |   |   vcs-git-commit.md
+|   |   |   ...
+|   |   |   vcs-personal-stories.md
+│   |   │   vcs-checklist.md
+│   |   │   vcs-resources.md
+|   └───example-chapter (new chapter folder)
+│       │   example-subchapter.md
+|       |   ...
+|       |   example-personal-stories.md
+│       │   example-checklist.md
+│       │   example-resources.md
+│   
+└───project-design <---- (folder for the Guide for Project Design) 
+    │   project-design.md
+    │   ...
+```
+
+To add more chapters to the Guide for Reproducible Research, 
+use the file locations of `example-chapter`'s constituent files for reference.
+
+(ch-content-checklist)=
+## Contribution Checklist
+
+### Follow the style and consistency guidelines
+
+As you write your chapter, keep _The Turing Way's_ [style](https://the-turing-way.netlify.app/community-handbook/style.html) and [consistency](https://the-turing-way.netlify.app/community-handbook/consistency.html) recommendations in mind.
+This ensures that your new content is accessible, and fits the overall style, structure, and formatting of the book.
+
+### Name files/folders appropriately
+
+Please follow _The Turing Way's_ conventions for naming files.
+With proper file names, other contributors can easily identify the purpose of your files and add to or improve them if necessary. 
+
+
+### Add your new files to the book's table of contents
+
+The book-wide table of contents lives in the `_toc.yml` [file](https://github.com/alan-turing-institute/the-turing-way/blob/master/book/website/_toc.yml).
+This file structures _The Turing Way_ and defines the order in which chapters appear.
+Your chapter's files should be added to the `_toc.yml` as appropriate.
+
+For example, because the [Statistical Methods Manuscript](https://the-turing-way.netlify.app/reproducible-research/case-studies/statistical-methods-manuscript.html) case study was written for the Guide for Reproducible Research, it was added to the table of contents as follows:
+
+
+```
+- file: reproducible-research/reproducible-research
+  sections:
+  ...
+  - title: Case Studies
+    file: reproducible-research/case-studies
+    sections:
+    - title: A Statistical Methods Manuscript
+      file: reproducible-research/case-studies/statistical-methods-manuscript
+```
+
+### Reference external sources appropriately
```

Comment:
Perhaps this checkpoint would be better placed after the maintaining consistency / following style guide checkpoint? You would do this as you wrap up the chapter and before you add it to the directory.